job-dsls/jobs/srcclr_scan_job.groovy

```diff
@@ -0,0 +1,81 @@
+import org.kie.jenkins.jobdsl.Constants
+
+def repoFile = new URL(Constants.REPO_FILE_URL).text
+def repoList = repoFile.readLines()
+
+for (repo in repoList) {
+
+    def jobName = "srcclr-scan-${repo}"
+
+    job(jobName) {
+
+        description("Job responsible for SourceClear verification of ${repo}")
+
+        parameters {
+            choiceParam('SCAN_TYPE', ['scm', 'binary'])
+            stringParam('SRCCLR_INVOKER_REPO_URL','')
+            stringParam('URL','')
+            stringParam('VERSION', '')
+            stringParam('PACKAGE','')
+            stringParam('NAME', '')
+            stringParam('MVNPARAMS', '')
+            choiceParam('PROCESSOR_TYPE', ['cve', 'cvss'])
+            booleanParam('RECURSIVE', false)
+            booleanParam('DEBUGGING', false)
+            booleanParam('TRACING', false)
+            stringParam('SCMVERSION', '')
+            stringParam('THRESHOLD', '1','Threshold from 1 to 10 for cvss processor')
+        }
+
```

Comment:
please always use the same separation type between pipeline blocks, we have zero, one and two lines separation types.

Reply:
Where is written?

Reply:
it's not written, my proposal is you just choose one in order to improve readability